README.md

```diff
@@ -0,0 +1,92 @@
+# Senior Game
```

Comment:
I would add information about the setup for the linux and windows build scripts or at least mention them.